src/app/components/FrostedGlassPromo/withData.tsx

```diff
@@ -164,7 +164,7 @@ const normalise = (props: PromoProps): FormattedPromo => {
 const validate = (props: FormattedPromo) => {
   try {
     return [props.children, props.image, props.url].every(Boolean);
-  } catch (e) {
+  } catch (_e) {
```

Comment:
The warning e is defined but never used occurs because the @typescript-eslint/no-unused-vars rule is flagging the unused error variable e in the catch block. Prefixing the variable with an underscore (_) indicates that it is intentionally unused, as allowed by the caughtErrorsIgnorePattern configuration in the ESLint setup.